.ci/release-and-publish.yml

```diff
@@ -1,10 +1,10 @@
 # Cross-platform set of build steps for building esy projects
 
 steps:
-  - script: esy x Oni2 --help
+  - script: esy @release x Oni2 --help
     displayName: "esy x Oni2 --help"
-  - script: esy create-release
-    displayName: "esy create-release"
+  - script: esy @release create
+    displayName: "esy @release create"
```

Comment:
We could use the `--codesign` flag here to get even more validation than we have today